nucypher/blockchain/eth/agents.py

```diff
@@ -356,69 +336,77 @@ def get_global_locked_tokens(self, at_period: int = None) -> int:
-        return self.contract.functions.lockedPerPeriod(at_period).call()
+        return NuNits(self.contract.functions.lockedPerPeriod(at_period).call())
 
     @contract_api(CONTRACT_CALL)
-    def get_staker_info(self, staker_address: str):
-        return self.contract.functions.stakerInfo(staker_address).call()
+    def get_staker_info(self, staker_address: ChecksumAddress) -> StakerInfo:
+        return StakerInfo(*self.contract.functions.stakerInfo(staker_address).call())
 
     @contract_api(CONTRACT_CALL)
-    def get_locked_tokens(self, staker_address: str, periods: int = 0) -> int:
+    def get_locked_tokens(self, staker_address: ChecksumAddress, periods: int = 0) -> NuNits:
         """
         Returns the amount of tokens this staker has locked
         for a given duration in periods measured from the current period forwards.
         """
         if periods < 0:
             raise ValueError(f"Periods value must not be negative, Got '{periods}'.")
-        return self.contract.functions.getLockedTokens(staker_address, periods).call()
+        return NuNits(self.contract.functions.getLockedTokens(staker_address, periods).call())
 
     @contract_api(CONTRACT_CALL)
-    def owned_tokens(self, staker_address: str) -> int:
+    def owned_tokens(self, staker_address: ChecksumAddress) -> NuNits:
         """
         Returns all tokens that belong to staker_address, including locked, unlocked and rewards.
         """
-        return self.contract.functions.getAllTokens(staker_address).call()
+        return NuNits(self.contract.functions.getAllTokens(staker_address).call())
 
     @contract_api(CONTRACT_CALL)
-    def get_substake_info(self, staker_address: str, stake_index: int) -> Tuple[int, int, int]:
+    def get_substake_info(self, staker_address: ChecksumAddress, stake_index: int) -> SubStakeInfo:
         first_period, *others, locked_value = self.contract.functions.getSubStakeInfo(staker_address, stake_index).call()
-        last_period = self.contract.functions.getLastPeriodOfSubStake(staker_address, stake_index).call()
-        return first_period, last_period, locked_value
+        last_period: Period = self.contract.functions.getLastPeriodOfSubStake(staker_address, stake_index).call()
+        return SubStakeInfo(first_period, last_period, locked_value)
 
     @contract_api(CONTRACT_CALL)
-    def get_raw_substake_info(self, staker_address: str, stake_index: int) -> Tuple[int, int, int, int]:
-        result = self.contract.functions.getSubStakeInfo(staker_address, stake_index).call()
+    def get_raw_substake_info(self, staker_address: ChecksumAddress, stake_index: int) -> RawSubStakeInfo:
+        result: RawSubStakeInfo = self.contract.functions.getSubStakeInfo(staker_address, stake_index).call()
         first_period, last_period, periods, locked = result
-        return first_period, last_period, periods, locked
+        return RawSubStakeInfo(first_period, last_period, periods, locked)
```

Comment:
Is it possible to return result here right after function call?

Reply:
I meant instead of `return RawSubStakeInfo(*result)` use `return result`, just curious

Reply:
:heavy_check_mark: